lib/huginn_agent/templates/newagent/newagent.gemspec.tt

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |spec|
 
   spec.files         = Dir['LICENSE.txt', 'lib/**/*']
   spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
-  spec.test_files    = Dir['spec/**/*.rb']
+  spec.test_files    = Dir['spec/{[!huginn/]}**/*.rb']
```

Comment:
I am afraid the bash glob syntax is not compatible to ruby's `Dir.glob`, this should work:
```ruby
Dir['spec/**/*.rb'].reject { |f| f[%r{^spec/huginn}]  }
```